api/src/main/java/au/org/aodn/nrmn/restapi/model/db/StagedRow.java

```diff
@@ -30,7 +31,7 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @JsonProperty(value = "Site No")
+    @JsonProperty(value = "site #")
```

Comment:
Wouldn't it be cleaner to just remove the ```@JsonProperty``` annotation completely? Same with site name and common name?

Reply:
I think i can remove the annotation for site name and common name